src/components/CheckboxInput/CheckboxInput.module.css

```diff
@@ -1,71 +1,67 @@
 @import '../../design-tokens/mixins.css';
 
-/*------------------------------------*\
-    # CHECKBOX INPUT
-\*------------------------------------*/
-
-/**
- * Wraps the visually hidden checkbox input element and the visible sibling svg element.
- */
-.input__wrapper {
-  position: relative;
-  /* Centers the checkbox icon in the wrapper. */
-  display: inline-flex;
-  align-items: center;
-  /* Aligns the checkbox with the first line of the label. */
-  align-self: flex-start;
-}
-/**
- * The visually hidden input element for the checkbox. The visual checkmark is provided by an svg element.
- */
 .checkbox__input {
-  cursor: pointer;
-  /* Removes default margins placed by browser for checkboxes. */
-  margin: 0;
-  /* Remove the checkbox from the page flow, positioning it on top of the SVG. */
-  position: absolute;
-  left: 0.25rem;
-  /* Set same dimensions as the CheckboxSvg element. */
-  height: var(--eds-size-2);
-  width: var(--eds-size-2);
-  /** 
-   * Hide the input element visually.
-   * This ensures the checkbox is still "physically" present so that all users,
-   * especially on touch screen readers, still interact with the real checkbox element
-   * where it would naturally be present. 
-   */
-  opacity: 0;
-}
+  height: 18px;
+  width: 18px;
+
+  /* The parent Checkbox component is a flex container. Make sure the input doesn't shrink. */
+  flex-shrink: 0;
+
+  /* Remove the browser's checkbox styles, allowing us to provide our own. */
+  appearance: none;
+
+  /* Magic value to center the checkbox on its label. */
+  margin: 3px;
```

Comment:
😐

Reply:
I think it's worth following up with design on this. Is this the only thing that is 18px square? and if so, how does that measure fit in with our sizes for other things. 

Setting a variable for just this doesn't help much, esp if it will only be used on this one line 🤔

Reply:
Agreed. This 3px is specific to the SVG. It was always there, but with this implementation is a bit more obvious. Definitely shouldn't make this a var.

Reply:
since we control all those px values, I wonder (per [Jin's comment](https://github.com/chanzuckerberg/edu-design-system/pull/1511/files#r1119519886)) if it's worth adding some `calc` math to at least make one of the numbers less magic. not a big deal though, things are well commented